dodola/cli.py

```diff
@@ -49,10 +49,21 @@ def dodola_cli(debug):
     required=True,
     help="URL to write NetCDF4 with adjusted simulation year to",
 )
-def apply_qdm(simulation, qdm, year, variable, out):
+@click.option(
+    "--include_quantiles",
+    "-q",
+    required=False,
+    help="Option to include simulation quantiles in output",
+)
```

Comment:
Can make this a simple flag with
```python
@click.option(
    "--include-quantiles",
    is_flag=True,
    help="Include simulation quantiles in output",
)
```
Feels more natural to me?

Reply:
I like this syntax idea - thanks!